src/pages/register.astro

```diff
@@ -0,0 +1,58 @@
+---
+import BaseLayout from "../layouts/BaseLayout.astro";
+import Hero from "../components/Hero.astro";
+import TextAndImage from "../components/TextAndImage.astro";
+import GroupPicture from "../img/GroupPicture.jpg";
+import VenueRainbow from "../img/VenueRainbow.png";
+import HealthSafetyShort from "../components/HealthSafetyShort.astro";
+import Tickets from "../components/Tickets.astro";
+---
+
+<style></style>
+<BaseLayout
+    pageTitle="Register for The Matrix Conference"
+    description="Join us for the Matrix Conference this October. The four day conference will take place in Strasbourg, France. Talks will be streamed live and virtual attendees will be able to ask speakers questions."
+>
+    <Hero
+        title="Get your tickets now"
+        description="Join us for the Matrix Conference this October. The four day conference will take place in Strasbourg, France. Talks will be streamed live and virtual attendees will be able to ask speakers questions."
```

Comment:
Can we get more flavor to the description? I believe this one is an outdated version of the one from /attend.